### PR TITLE
4 add blog functionality

### PR DESCRIPTION
**Pull request by xarrijorge**
In summary:
       Setup basic blog functionality using @nuxt/content@^1 - Completed
       Design simple page showing individual blogs - Completed
       Add /blog/ route listing blogs - Completed
       Consider adding blog entries to the home page. - Completed

### CODE REVIEW

**Comment by markbrough** [comment]
OK I built a site here for now:
https://humportal.github.io/humportal-dev/

(still need to work on the workflow for that; I just merged to dev on the command line for now and then pushed to the dev branch)

This is looking great already! Here is some feedback to tweak the style a bit. I have shared the template with colleagues to fill them out for all the blogs etc. that we want to have.

Home page:
* Let's change "Featured Blogs" to "Updates" as this is sufficiently broad a term to use for both blogs and featured items
* Change it also on the navbar
* Explore data -> change to "Read more" and link to the update page
* Instead of cutting the text off, I think we should have a short excerpt -- **I have pushed two commits** adding a new `summary` property and moving the rest of the text to the body of the markdown file. It's better to have this information here rather than in a frontmatter property because we can then use all the features of markdown in the updates (e.g. adding links etc.)

Blogs page:
* This page looks great.
* It's great that you used the `blogList.vue` component! Makes things super simple :))
* Will get you some text for the top!

Individual blog page:
* We don't have an image always... the `` element should only be shown if we have an image.
* Let's change the container from `<b-jumbotron>` to `<b-container>` because the grey is not very easily readable. We also don't need to see the page slug printed on the page.
* Please add a link to the url as a button.
  * For blogs, let's label this "Read more"
  * For features, let's label this "Visit the site"
  * In both cases, I think it would be nice to add the Font Awesome icon to the button: https://fontawesome.com/icons/arrow-up-right-from-square?s=solid&f=classic

**Comment by xarrijorge** [comment]
> OK I built a site here for now: https://humportal.github.io/humportal-dev/
> 
> (still need to work on the workflow for that; I just merged to dev on the command line for now and then pushed to the dev branch)
> 
> This is looking great already! Here is some feedback to tweak the style a bit. I have shared the template with colleagues to fill them out for all the blogs etc. that we want to have.
> 
> Home page:
> 
> * Let's change "Featured Blogs" to "Updates" as this is sufficiently broad a term to use for both blogs and featured items
> * Change it also on the navbar
> * Explore data -> change to "Read more" and link to the update page
> * Instead of cutting the text off, I think we should have a short excerpt -- **I have pushed two commits** adding a new `summary` property and moving the rest of the text to the body of the markdown file. It's better to have this information here rather than in a frontmatter property because we can then use all the features of markdown in the updates (e.g. adding links etc.)
> 
> Blogs page:
> 
> * This page looks great.
> * It's great that you used the `blogList.vue` component! Makes things super simple :))
> * Will get you some text for the top!
> 
> Individual blog page:
> 
> * We don't have an image always... the `` element should only be shown if we have an image.
> * Let's change the container from `<b-jumbotron>` to `<b-container>` because the grey is not very easily readable. We also don't need to see the page slug printed on the page.
> * Please add a link to the url as a button.
>   
>   * For blogs, let's label this "Read more"
>   * For features, let's label this "Visit the site"
>   * In both cases, I think it would be nice to add the Font Awesome icon to the button: https://fontawesome.com/icons/arrow-up-right-from-square?s=solid&f=classic

All suggestions have been implemented.

**Review by markbrough — CHANGES_REQUESTED** [review]
Looks great! One tiny tweak to the width of the container for the mobile view and a couple of other suggestions to remove stray comments

Then: the one remaining thing we need before merging this into `main` is the blog / feature content -- I am working on getting that together!

File: components/blogList.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<template>
+    <b-row style="margin-top: 50px" align-h="center">
+        <b-card-group v-for="blog of dataList" :key="blog.slug" column>
+            <!-- <NuxtLink :to="`blogs/${blog.slug}`"> -->
```
[the remark this answers is not in the document]
Comment (xarrijorge) [reply components/blogList.vue:4]: All changes implemented!

**Review by xarrijorge — COMMENTED** [review]
All changes have been implemented

**Comment by markbrough** [comment]
Hey @xarrijorge thanks for all your work on this! I added a bunch of blog / features content and made some stylistic tweaks. If you are happy with this, please feel free to go ahead and merge!

**Review by markbrough — APPROVED** [review]

**Pull request merged**